ui/lib/config-ui/addon/components/messages/message-expiration-date-form.hbs

```diff
@@ -0,0 +1,54 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+<div class="has-top-padding-xs has-bottom-padding-s is-narrow is-flex-align-baseline">
+  <RadioButton
+    class="radio"
+    name={{@attr.name}}
+    id="never"
+    value="never"
+    @value="never"
+    @onChange={{fn (mut @message.endTime) ""}}
+    @groupValue={{this.groupValue}}
+  />
+  <div class="has-left-margin-xs">
+    <label for="never" class="has-left-margin-xs has-text-black is-size-7">
+      Never
+      <p class="has-left-margin-xs has-text-grey is-size-8">
+        This message will never expire unless manually deleted by an operator.
+      </p>
+    </label>
+  </div>
+</div>
+
+<div class="has-top-padding-xs has-bottom-padding-s is-narrow is-flex-align-baseline">
+  <RadioButton
+    class="radio"
+    name={{@attr.name}}
+    id="specific-date"
+    value="specificDate"
+    @value="specificDate"
+    @onChange={{(fn (mut this.groupValue) "specificDate")}}
+    @groupValue={{this.groupValue}}
+  />
+  <div class="has-left-margin-xs">
```

Comment:
nit: do we need this wrapping div, or could we just add a larger left margin to the label?